demo-forms/visible-everywhere.context.json

```diff
@@ -0,0 +1,3 @@
+{
+	"expression": "true"
+}
```

Comment:
Currently the form upload in Config requires a JSON context file.  Also if you don't know it's the default (as I did not), you might like to explicitly set it.